tests/func/experiments/test_init.py

```diff
@@ -136,13 +136,19 @@ def test_init_interactive_when_no_path_prompts_need_to_be_asked(
             "train": {
                 "cmd": "cmd",
                 "deps": ["data", "src"],
-                "live": {"dvclive": {"html": True, "summary": True}},
-                "metrics": [{"metrics.json": {"cache": False}}],
-                # we specify `live` through `overrides`,
-                # so it creates checkpoint-based output.
-                "outs": [{"models": {"checkpoint": True}}],
+                "metrics": [
+                    {"dvclive.json": {"cache": False}},
+                    {"metrics.json": {"cache": False}},
+                ],
+                # TODO: discuss:
+                # IMO, specifying `live` should not create checkpoint output
+                # unless type="dl" is passed
+                "outs": ["models"],
```

Comment:
@dberenbaum / @skshetry I might have missed the discussion about this behavior.

Enabling checkpoints makes sense with `type="dl"`, but passing `live` should not mean that `checkpoints` are also enabled. 

I have changed the previous behavior in order to decouple `live` from `checkpoints`. Checkpoints are only enabled with `type="dl"`.

Reply:
It was introduced in https://github.com/iterative/dvc/pull/6716. I don't remember exactly why it was done this way, but probably it was done to unify with interactive mode (even with `--type=dl`, users can omit `live`).

It makes sense to do that anyway on `--type="dl"`.